twingate/internal/test/acctests/datasource/groups_test.go

```diff
@@ -176,22 +176,31 @@ func TestAccDatasourceTwingateGroups_WithEmptyFilters(t *testing.T) {
 			},
 			Steps: []resource.TestStep{
 				{
-					Config:             testTwingateGroupsWithEmptyFilter(),
+					Config:             testTwingateGroupsWithEmptyFilter(test.RandomGroupName()),
 					ExpectNonEmptyPlan: true,
+					Check: acctests.ComposeTestCheckFunc(
+						testCheckResourceAttrNotEqual("data.twingate_groups.all", groupsLen, "0"),
```

Comment:
Why would it be empty with no filters?  It should return all...

Reply:
I'm expecting at least one group, but we might have more on the test tenant